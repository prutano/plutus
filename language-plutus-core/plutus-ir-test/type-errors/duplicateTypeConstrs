(let
  (nonrec)
   (datatypebind
    (datatype
      (tyvardecl Either (fun (type) (fun (type) (type))))
      (tyvardecl a (type))
      (tyvardecl a (type))
      match_Either
      (vardecl Right (fun a [Either a a]))
      (vardecl Left (fun a [Either a a]))
    )
  )
  (con 5)
)
