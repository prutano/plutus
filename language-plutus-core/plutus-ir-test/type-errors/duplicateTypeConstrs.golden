Error during typechecking:
A duplicate identifier is introduced inside a single datatype-bind of let at duplicateTypeConstrs:4:6
Offending identifier: a